Example/Podfile

```diff
@@ -272,3 +261,14 @@ target 'InstanceID_Example_tvOS' do
   end
 end
 
+# Set define to turn on GUL_UNSWIZZLING and GUL_APP_DELEGATE_TESTING for the unit tests
+post_install do |installer_representation|
```

Comment:
Now (Cocoapods 1.6.1) there is only `post_install` hook supported for entire Podfile. It affects all targets anyways, so I moved it to the upper level to avoid confusion.

Reply:
Cool, thanks!